src/js/models/appUser.js

```diff
@@ -13,7 +13,9 @@ var AppUser = module.exports = Backbone.AssociatedModel.extend({
     },
 
     parse: function(data) {
-        return data.appUser;
+        var appUser = data.appUser;
+        appUser.signedUpAt = new Date(data.appUser.signedUpAt);
```

Comment:
to be more specific, `Date` gets serialized to a ISO 8601 string on the wire, so unless we do this we get back a string in the backbone model when we sync. I want it to be a `Date` going in and a date coming back out again.